gnovm/cmd/gno/lint.go

```diff
@@ -49,6 +52,31 @@ func (c *lintCfg) RegisterFlags(fs *flag.FlagSet) {
 	fs.IntVar(&c.setExitStatus, "set-exit-status", 1, "set exit status to 1 if any issues are found")
```

Comment:
It's used in tests; specifically because we use `os.Exit` in `lint.go` and that will also quit tests

I'm thinking of adding a `type ErrExit int` to `pkg/commands`, so instead of using os.Exit here we can `return commands.ErrExit(1)` instead. Wdyt?

This would remove the need for the `--set-exit-status` and make the behaviour testable